tensorflow/lite/micro/kernels/lstm_eval.h

```diff
@@ -95,134 +89,162 @@ struct IntegerLstmParameter {
-    const TfLiteTensor* cell_to_input_weights,
-    const TfLiteTensor* cell_to_forget_weights,
-    const TfLiteTensor* cell_to_output_weights,
-    const TfLiteTensor* input_layer_norm_coefficients,
-    const TfLiteTensor* forget_layer_norm_coefficients,
-    const TfLiteTensor* cell_layer_norm_coefficients,
-    const TfLiteTensor* output_layer_norm_coefficients,
-    const TfLiteTensor* aux_input,
-    const TfLiteTensor* aux_input_to_input_weights,
-    const TfLiteTensor* aux_input_to_forget_weights,
-    const TfLiteTensor* aux_input_to_cell_weights,
-    const TfLiteTensor* aux_input_to_output_weights,
-    const TfLiteTensor* input_gate_bias, const TfLiteTensor* forget_gate_bias,
-    const TfLiteTensor* cell_gate_bias, const TfLiteTensor* output_gate_bias,
-    const TfLiteTensor* projection_weights, const TfLiteTensor* projection_bias,
-    const TfLiteLSTMParams* params, bool forward_sequence, bool time_major,
-    int output_offset, TfLiteTensor* scratch_buffer, TfLiteTensor* output_state,
-    TfLiteTensor* cell_state, TfLiteTensor* output);
-
-TfLiteStatus EvalHybrid(
-    const TfLiteTensor* input, const TfLiteTensor* input_to_input_weights,
-    const TfLiteTensor* input_to_input_weights_ledger,
-    const TfLiteTensor* input_to_forget_weights,
-    const TfLiteTensor* input_to_forget_weights_ledger,
-    const TfLiteTensor* input_to_cell_weights,
-    const TfLiteTensor* input_to_cell_weights_ledger,
-    const TfLiteTensor* input_to_output_weights,
-    const TfLiteTensor* input_to_output_weights_ledger,
-    const TfLiteTensor* recurrent_to_input_weights,
-    const TfLiteTensor* recurrent_to_input_weights_ledger,
-    const TfLiteTensor* recurrent_to_forget_weights,
-    const TfLiteTensor* recurrent_to_forget_weights_ledger,
-    const TfLiteTensor* recurrent_to_cell_weights,
-    const TfLiteTensor* recurrent_to_cell_weights_ledger,
-    const TfLiteTensor* recurrent_to_output_weights,
-    const TfLiteTensor* recurrent_to_output_weights_ledger,
-    const TfLiteTensor* cell_to_input_weights,
-    const TfLiteTensor* cell_to_forget_weights,
-    const TfLiteTensor* cell_to_output_weights,
-    const TfLiteTensor* input_layer_norm_coefficients,
-    const TfLiteTensor* forget_layer_norm_coefficients,
-    const TfLiteTensor* cell_layer_norm_coefficients,
-    const TfLiteTensor* output_layer_norm_coefficients,
-    const TfLiteTensor* aux_input,
-    const TfLiteTensor* aux_input_to_input_weights,
-    const TfLiteTensor* aux_input_to_forget_weights,
-    const TfLiteTensor* aux_input_to_cell_weights,
-    const TfLiteTensor* aux_input_to_output_weights,
-    const TfLiteTensor* input_gate_bias, const TfLiteTensor* forget_gate_bias,
-    const TfLiteTensor* cell_gate_bias, const TfLiteTensor* output_gate_bias,
-    const TfLiteTensor* projection_weights,
-    const TfLiteTensor* projection_weights_ledger,
-    const TfLiteTensor* projection_bias, const TfLiteLSTMParams* params,
+// Scales for hybrid op
```

Comment:
nit: do you mind add a short comment here or somewhere else what hybrid means?

Reply:
Added.